src/ext/io.rs

```diff
@@ -144,5 +144,17 @@ where
     Report<E>: From<E>,
     F: FnOnce() -> Result<T, E> + Send + 'static,
 {
-    run_background(|| work().into_report()).await
+    spawn_blocking_stacked(|| work().into_report()).await
+}
+
+/// Run the provided blocking closure in the background,
```

Comment:
I don't understand when I would use `spawn_blocking` and when I would use `spawn_blocking_stacked`, and their comments are identical.  Can we make that more clear?

Reply:
The difference is in the types: `spawn_blocking` changes the error to an `error_stack::Report`, while `spawn_blocking_stacked` expects an existing `error_stack::Report`.

I'll definitely disambiguate these before merging.